pentasharp/Models/Entities/TaxiCompany.cs

```diff
@@ -23,5 +23,6 @@ public class TaxiCompany
 		public DateTime? UpdatedAt { get; set; }
 
 		public ICollection<Taxi> Taxis { get; set; } = new List<Taxi>();
-	}
+        public bool IsDeleted { get; set; }
```

Comment:
leave one space in between properties and remove data annotations since we agreed on not using them like this anymore

ctrl + a , ctrl + k + f should format the code